orttraining/orttraining/training_ops/cpu/torch/torch_custom_function_kernel_base.h

```diff
@@ -149,6 +149,8 @@ class PythonOpBase {
   // Output types of MyReLU.apply(...).
   std::vector<int64_t> output_tensor_types_;
 
+  bool safe_run_mode_enabled_{true};
```

Comment:
What is this? Does runner own input tensors if this is true? If this is false, the runner doesn't change refcnt?

Reply:
Explained in the above comment. Safe mode will be the original and default behaviour. For autograd functions, especially for those defined by ourselves, if they don't use ctx for torch APIs, it is ok to use unsfafe mode, to get some trade off of perfomrance.